server/job_controller/api_services/audio_process.py

```diff
@@ -9,12 +9,24 @@
 from google.cloud.speech import enums
 from google.cloud.speech import types
 
+
+# Set up Google Custom Search for identifying movies
+from dotenv import load_dotenv
+import requests
+import collections
+
+load_dotenv()
+API_KEY = os.getenv('CUSTOM_SEARCH_API_KEY')
+CSE_ID = os.getenv('CUSTOM_SEARCH_ID')
+
+base_url = "https://www.googleapis.com/customsearch/v1"
+
 def ExtractAudio(video_path):
-    outputPath = "../audio/Inception_Audio.mp3"
+    fileName = re.search("(.*\/|^)(.*?)\.mp4", video_path).group(2)
+    outputPath = f"../audio/{fileName}.mp3"
```

Comment:
we should look to keep all the processing files within the same directory as the original video just to keep the directories clean (and easy to work with)